lib/problems.rb

```diff
@@ -1,9 +1,40 @@
 require_relative './stack.rb'
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(n)
 def balanced(string)
```

Comment:
👍 Instead of using the `case` statements, you could instead use a hash where the keys are the close braces and the values are the matching open braces.